shared/src/commonMain/kotlin/co/nimblehq/ic/kmm/suv/domain/model/Question.kt

```diff
@@ -23,6 +23,12 @@ data class Question(
 ) {
     val sortedAnswers: List<Answer>
         get() = answers.sortedBy { it.displayOrder }
+
+    val answerTexts: List<String>
+        get() = sortedAnswers.map { it.text ?: "" }
+
+    val answerPlaceholders: List<String>
+        get() = sortedAnswers.map { it.inputMaskPlaceholder ?: "" }
```

Comment:
`?: ""` -> `.orEmpty()`

Reply:
we need to use https://kotlinlang.org/api/latest/jvm/stdlib/ to have extra extensions like this, it's up to @suho imo.

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/121/commits/65286cf9f85f749815db7767583f78ed841ced98

Reply:
> we need to use [kotlinlang.org/api/latest/jvm/stdlib](https://kotlinlang.org/api/latest/jvm/stdlib/) to have extra extensions like this

@luongvo Just curious, does it affect anything? It seems there is no import

Reply:
@suho we need to import `const val STDLIB = "org.jetbrains.kotlin:kotlin-stdlib:${Versions.KOTLIN}"` to commonMain 😛

Reply:
LOL, but I did not add anything, just write `orEmpty()`